app/src/main/java/org/literacyapp/authentication/AuthenticationActivity.java

```diff
@@ -123,6 +130,8 @@ public void onCameraViewStopped() {
     public Mat onCameraFrame(CameraBridgeViewBase.CvCameraViewFrame inputFrame) {
         Mat imgRgba = inputFrame.rgba();
 
+        DetectionHelper.adjustScreenBrightness(getApplicationContext(), imgRgba);
```

Comment:
I though the same. When I execute Settings.System.getInt I am forced to have a try/catch block but for Settings.System.putInt there is no such thing. Hmmmm..